cli/cmd/sudo/user/assume.go

```diff
@@ -18,6 +21,22 @@ func AssumeCmd(ch *cmdutil.Helper) *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ctx := cmd.Context()
 
+			representingUser, err := dotrill.GetRepresentingUser()
+			if err != nil {
+				ch.PrintfWarn("Could not parse representing user email\n\n")
```

Comment:
Include the `err` in the error log for easier debugging
```suggestion
				ch.PrintfWarn("Could not parse representing user email: %v\n\n", err)
```